test/compiler.ts

```diff
@@ -68,10 +69,10 @@ function runTests (solc, versionText) {
     }
   }
 
-  function expectError (output, errorType, message) {
+  function expectError (output: any, errorType: any, message: any) {
```

Comment:
Do you plan to strengthen the types later for this? I think that is preferred, but curious now.

Reply:
Yes, that seems to be the plan.

> ### What's next?
>
> Once merged, this allows the introduction of typing to reduce errors and improve code quality. This can go in hand with enabling typescript declaration and configuration of the package.json to use said built types. Allowing users of the application to gain access to these types after installing the package.